### DISCUSSION TITLE
Running Script in Sql Server

### DISCUSSION THREAD

**Discussion by sobi1995** [dr5hn/countries-states-cities-database#194] [Ideas]
when I want to execute a script in SQL server I get some errors! are they different? MySQL and SQL server I mean

**Comment by dr5hn** [comment]
@Sobi1995 I think so !! But what are the errors?

**Comment by sobi1995** [comment]
they are syntax errors!
what can I do?

**Comment by dr5hn** [comment]
@Sobi1995 Can you post them here? So that I can check and if possible we can fix it.

**Comment by bregoh** [comment]
@Sobi1995 the sql file is mostly compatible with mysql server, hence the syntax error. 

My suggestion is you use the csv file or json file using programming code to update the database. If you need assistance let me know.

**Comment by dr5hn** [comment]
@Sobi1995 //cc @bregoh  I want to make this Database compatible with SQL Server as well.

**Comment by sobi1995** [comment]
> @Sobi1995 Can you post them here? So that I can check and if possible we can fix it.

![image](https://user-images.githubusercontent.com/31405858/102183219-b8cdba80-3ec2-11eb-8f8a-150f6272d846.png)

**Comment by sobi1995** [comment]
> @Sobi1995 the sql file is mostly compatible with mysql server, hence the syntax error.
> 
> My suggestion is you use the csv file or json file using programming code to update the database. If you need assistance let me know.

thanks
I thought the same

**Comment by dr5hn** [comment]
@Sobi1995 Thanks for the screenshot. If you can wait, I will fix these issues by this weekend and make it SQL Server compatible.

**Comment by sobi1995** [comment]
> @Sobi1995 Thanks for the screenshot. If you can wait, I will fix these issues by this weekend and make it SQL Server compatible.
thanks 
I wanted to fix that and send a pull request but I wait for you

**Comment by dr5hn** [comment]
@Sobi1995 

In the meantime, If you want to fix it, please go ahead.

You can use this site for syntax comparison
http://www.sqlines.com/sql-server-to-mysql

If you've got a windows machine, You can also download the tool for conversion, you need to just put our SQL script and click on convert.
http://www.sqlines.com/download
